test/test_nn.py

```diff
@@ -5061,8 +5061,9 @@ def test_hardtanh_backward(self):
         x_grad_ref = torch.where(mask, grad, z)
         self.assertEqual(x.grad, x_grad_ref)
 
-    def test_batchnorm_nhwc_cpu(self):
-        def helper(self, mod, size, dtype, mixed_dtype=False, format=torch.channels_last):
+    # Temporary renaming for CI test
+    def test_batchnorm_nhwc_rename(self):
```

Comment:
To fix a flaky test issue, I think opening a PR with "Fixes <flaky test issue number>" in the initial commit message would make the test run in CI. In this case, since you went via the renaming method, I think we can rename this back and CI bot will automatically close the flaky test issue after this PR is landed when it detects that the test is no longer flaky